### PR TITLE
Fixed addClass fallback path, className was not passed into it

### PR DESCRIPTION
**Pull request by Andarist**

### CODE REVIEW

File: test/class.js [hunk 1]
```diff
@@ -15,6 +20,22 @@ describe('Class helpers', () => {
     expect(el.className).to.contain('my-class')
   })
 
+  it('should add a class properly when using a fallback path', () => {
+    var el = document.getElementById('item-1')
+    removeProperty('classList', el)
```
Comment (jquense) [inline test/class.js]: Can you reset this after the test?

Comment (Andarist) [inline test/class.js]: I can, but this change is only on the instance and not on the prototype itself and u make fresh nodes from fixtures in beforeEach hook anyway.

Comment (jquense) [inline test/class.js]: Fair point

**Comment by jquense** [comment]
Thanks!

**Pull request merged**